docs/DeveloperGuide.md

```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
           Use case ends.
 
+**Use case: Check eligibility for assistance**
+
+**MSS:**
+
+1. User chooses which person to check eligibility for assistance
+2. SocialBook displays what schemes the person is eligible for
+
+    Use case ends.
+
+**Extensions:**
+
+* 1a. Given index is invalid.
+    * 1a1. SocialBook informs user that the index is invalid
+      Use case resumes at step 1.
+
+
+**Use case: Add a scheme to a person**
+
+**MSS**
+1. User checks what schemes a person is eligible for.
```

Comment:
Think can remove this point as I think MSS would assume the user already looked at eligible schemes and knows which one they r trying to add to the person

Reply:
im using this to reference the mentioned use case above